test/test_metar.py

```diff
@@ -513,8 +521,8 @@ def report(runway_state):
     assert report("09690692 27550591").temp.value() == -1.0
     assert report("09690692 27550591").remarks() == ""
 
-    assert report("09SNOCLO").remarks() == ""
-    assert report("09CLRD//").remarks() == ""
```

Comment:
Would these tests now fail?

Reply:
It would appear so, this makes me somewhat nervous about this PR.

Reply:
You OK with this now @phobson ?